x-pack/test/functional/apps/canvas/smoke_test.js

```diff
@@ -61,7 +61,7 @@ export default function canvasSmokeTest({ getService, getPageObjects }) {
         // check that the elements are what we expect
 
         // first is a markdown element
-        const md = await elements[0].findByCssSelector('.canvasMarkdown');
+        const md = await elements[0].findByCssSelector('.kbnMarkdown__body');
```

Comment:
This change isn't required if you restore the `canvasMarkdown` class on the markdown renderer.